packages/charts/theme/lumo/vaadin-chart-styles.js

```diff
@@ -0,0 +1,93 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
+import '@vaadin/vaadin-lumo-styles/color.js';
```

Comment:
This import needs to be placed before the named import above.

Reply:
Done.